ext/fetch/lib.rs

```diff
@@ -419,17 +573,42 @@ pub async fn op_fetch_response_read(
     .resource_table
     .get::<FetchResponseBodyResource>(rid)?;
   let mut reader = RcRef::map(&resource, |r| &r.reader).borrow_mut().await;
-  let cancel = RcRef::map(resource, |r| &r.cancel);
+  let cancel = RcRef::map(&resource, |r| &r.cancel);
   let mut buf = data.clone();
   let read = reader.read(&mut buf).try_or_cancel(cancel).await?;
+
+  // Update the bytes read for DevTools
+  resource.bytes_read.fetch_add(read as u32, Ordering::SeqCst);
+  let mut request_bodies =
+    resource.dev_tools_agent.request_bodies.lock().unwrap();
+  if let Some(body) = request_bodies.get_mut(&resource.id) {
+    body.extend_from_slice(&buf[..read])
+  }
```

Comment:
This currently never frees the memory, add a handler that drops all bodies if all subscribers disconnect